function/builtin/transform/special.go

```diff
@@ -140,8 +140,8 @@ var ExponentialMovingAverage = function.MakeFunction(
 
 // Alias is deprecated.
```

Comment:
Okay- any queries using it will have to be updated, because they'll start to error